proposals/lip-0063.md

```diff
@@ -298,6 +298,7 @@ Some new LIPs introduced configurable constants. The values for these constants
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `NUMBER_STANDBY_VALIDATORS` | 2 |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `TOKEN_ID_POS` | `TOKEN_ID_LSK` |
 | [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `VALIDATOR_REGISTRATION_FEE` | 10 x (10^8) |
+| [PoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants) | `INVALID_BLS_KEYS_IN_GENESIS_BLOCK` | `True` |
```

Comment:
The specifics on how to use the constant `INVALID_BLS_KEYS_IN_GENESIS_BLOCK` within the migration tool/logic is missing. Could you please update the same?

Reply:
The migrator tool does not need to use this constant. It is only needed by Lisk Core v4. More precisely, the PoS module needs this config value for the genesis block processing.